.solcover.js

```diff
@@ -10,6 +10,6 @@ module.exports = {
     gasLimit: 0xfffffffffff,
   },
   mocha: {
-    enableTimeouts: false
+    enableTimeouts: true
```

Comment:
Is this intentional?

Reply:
yes some tests were getting stuck during testing